.vscode/cspell.json

```diff
@@ -274,6 +274,7 @@
     "embedme",
     "encryptor",
     "endtoend",
+    "entra",
```

Comment:
Should this entry be `Entra` and case enforced?

@jairmyree do we have case sensitivity enforced in our cspell configuration - https://cspell.org/docs/case-sensitive/

Reply:
@joshfree Our cspell config does not have case-sensitivity enabled. Casing is not enforced.